apps/ejabberd/src/ejabberd_c2s.erl

```diff
@@ -2002,18 +2002,6 @@ presence_broadcast_first(From, StateData, Packet) ->
             StateData#state{pres_a = As}
     end.
 
-
--spec remove_element(E :: 'error' | tuple(),
-                     Set :: gb_set()) -> gb_set().
-remove_element(E, Set) ->
-    case ?SETS:is_element(E, Set) of
```

Comment:
Ups, sorry, somehow this was not marked as removed line, or it's just me.... Any way, no more comments on this :)